content/go/concepts/math-functions/terms/floor/floor.md

```diff
@@ -0,0 +1,80 @@
+---
+Title: 'Floor()'
+Description: 'Returns the given decimal number rounded down to the nearest lower value.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Floor()`** function returns a given decimal number rounded down to the nearest lower value.
+
+## Syntax
+
+```pseudo
+math.Floor(decimalNumber float64) float64
+```
+
+The `Floor()` function accepts a single parameter of type float64 and rounds it down to the nearest lower value.
+Take notice that the returned value may appear to be an `int`, but it is a `float64` value.
+
+In some exceptional cases, passing any of the following values: `±Inf`, `±0`, and `NaN` as an argument to the `Floor()` function, will result in the returned value being the same as the argument.
+
+## Example
+
```

Comment:
There should be a short introductory sentence here

Reply:
@caupolicandiaz You mean for the Example part ?